sites/public/src/layouts/application.tsx

```diff
@@ -1,10 +1,9 @@
 import React from "react"
 import { useRouter } from "next/router"
-import Link from "next/link"
 import Head from "next/head"
-import { SiteFooter, FooterNav, FooterSection, t } from "@bloom-housing/ui-components"
-import { ExygyFooter } from "@bloom-housing/shared-helpers"
 import { getSiteHeader } from "../lib/helpers"
+import { t } from "@bloom-housing/ui-components"
+import { SiteFooter, FooterNav, FooterSection } from "@bloom-housing/doorway-ui-components"
```

Comment:
(not blocking, not actionable, also perhaps not a correct opinion just thinking out loud) I wonder if this could have been better implemented as a separate component like DoorwaySiteFooter vs. reusing the SiteFooter.  

* They don't look very similar
* The CSS override thing is annoying to need to code around
* Basically as-implemented we use SiteFooter as a shell and then write all of the JSX inline anyway

Reply:
Yeah I started wondering that too with the CSS override issues, it probably would have been easier to do it that way from the start